community-operators/kubevirt/kubevirtoperator.0.17.0-alpha.0.csv.yaml

```diff
@@ -0,0 +1,565 @@
+              verbs:
+              - create
+              - patch
+            - apiGroups:
+              - ""
+              resources:
+              - configmaps
+              verbs:
+              - get
+              - list
+              - watch
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - version
+              verbs:
+              - get
+              - list
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachineinstances/console
+              - virtualmachineinstances/vnc
+              verbs:
+              - get
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachines/restart
+              verbs:
+              - put
+              - update
+            - apiGroups:
+              - kubevirt.io
+              resources:
+              - virtualmachines
+              - virtualmachineinstances
+              - virtualmachineinstancepresets
+              - virtualmachineinstancereplicasets
+              verbs:
+              - get
+              - delete
+              - create
+              - update
+              - patch
+              - list
+              - watch
+              - deletecollection
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachineinstances/console
+              - virtualmachineinstances/vnc
+              verbs:
+              - get
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachines/restart
+              verbs:
+              - put
+              - update
+            - apiGroups:
+              - kubevirt.io
+              resources:
+              - virtualmachines
+              - virtualmachineinstances
+              - virtualmachineinstancepresets
+              - virtualmachineinstancereplicasets
+              verbs:
+              - get
+              - delete
+              - create
+              - update
+              - patch
+              - list
+              - watch
+            - apiGroups:
+              - kubevirt.io
+              resources:
+              - virtualmachines
+              - virtualmachineinstances
+              - virtualmachineinstancepresets
+              - virtualmachineinstancereplicasets
+              verbs:
+              - get
+              - list
+              - watch
+            - apiGroups:
+              - authentication.k8s.io
+              resources:
+              - tokenreviews
+              verbs:
+              - create
+            - apiGroups:
+              - authorization.k8s.io
+              resources:
+              - subjectaccessreviews
+              verbs:
+              - create
+
+      deployments:
+        - name: virt-operator
+          spec:
+            replicas: 2
```

Comment:
@aneeshkp  I think this is what @AlexNPavel is referring to as the cause for the error in CI.
Does this operator need 2 replicas to work? Would it be possible to set the replicas to 1?

Reply:
changed replica to 1